tools/t1k.cwl

```diff
@@ -9,30 +9,62 @@ requirements:
-    inputBinding: { position: 2 , prefix: "--preset"} }
-  threads: { type: 'int?', doc: "Num processing threads to use", default: 8, inputBinding: { position: 2, prefix: "-t" } }
+  t1k_ref_tar: { type: File, doc: "Tar ball with T1K reference", inputBinding: { position: 2 } }
+  t1k_ref_path: { type: string, doc: "String of path to un-tarred hla ref, i.e. kiridx/kiridx_rna_seq.fa", inputBinding: { position: 12, prefix: "-f"}  }
+
+  reads: { type: 'File?', doc: "read1 fastq reads if paired-end", inputBinding: { position: 12 , prefix: "-1" } }
+  mates: { type: 'File?', doc: "mates (read2) fastq reads if paired-end", inputBinding: { position: 12 , prefix: "-2" } }
+  single_end: { type: 'File?', doc: "fastq reads if single-end", inputBinding: { position: 12 , prefix: "-u" } }
+  interleaved: { type: 'File?', doc: "fastq reads if interleaved", inputBinding: { position: 12 , prefix: "-i" } }
+  bam: { type: 'File?', doc: "if bam input", inputBinding: { position: 12 , prefix: "-b" } }
+  output_basename: { type: 'string?', doc: "Prefix string for output file names. Default inferred from input", inputBinding: { position: 12, prefix: "-o"} }
+  preset: {type: ['null', {type: enum, name: preset, symbols: ["hla", "hla-wgs", "kir-wgs", "kir-wes"]}], default: "hla", doc: "If paired-end, read orientation", inputBinding: { position: 12 , prefix: "--preset"} }
+  stage: {type: ['null', {type: enum, name: stage, symbols: ["0", "1", "2"]}], inputBinding: { position: 12 , prefix: "--stage" }, doc: "start genotyping on specified stage; 0: start from beginning (candidate read extraction). 1: start from genotype with candidate reads. 2: start from post analysis" }
+
+  gene_coordinates: { type: 'File?', inputBinding: { position: 12 , prefix: "-c" }, doc: "gene coordinate file (required when providing BAM/-b input)" }
+  min_align_sim: { type: 'float?', inputBinding: { position: 12 , prefix: "-s" }, doc: "minimum alignment similarity" }
+  frac: { type: 'float?', inputBinding: { position: 12 , prefix: "--frac" }, doc: "filter if abundance is less than the frac of dominant allele" }
+  cov: { type: 'float?', inputBinding: { position: 12 , prefix: "--cov" }, doc: "filter genes with average coverage less than the specified value" }
+  cross_gene_rate: { type: 'float?', inputBinding: { position: 12 , prefix: "--crossGeneRate" }, doc: "the effect from other gene's expression" }
+  allele_digit_units: { type: 'int?', inputBinding: { position: 12 , prefix: "--alleleDigitUnits" }, doc: "the number of units in genotyping result" }
+  allele_delimiter: { type: 'string?', inputBinding: { position: 12 , prefix: "--alleleDelimiter" }, doc: "the delimiter character for digit unit" }
+  allele_whitelist: { type: 'string?', inputBinding: { position: 12 , prefix: "--alleleWhitelist" }, doc: "only consider read aligned to the listed allele sereies" }
```

Comment:
Typo in series

allele_whitelist: { type: 'string?', inputBinding: { position: 12 , prefix: "--alleleWhitelist" }, doc: "only consider read aligned to the listed allele series" }

Reply:
Got it